src/helpers/Blocks.d.ts

```diff
@@ -38,16 +40,13 @@ export interface Pow {
     pow_data: string;
 }
 export interface Body {
-    inputs: InputsEntity[];
-    outputs: OutputsEntity[];
-    kernels: KernelsEntity[];
+    inputs: Inputs[];
+    outputs: Outputs[];
+    kernels: Kernels[];
```

Comment:
These must stay the same, the extra fields are irrelevant for the rest of the system, only use the extended ones in the SingleBlock
```suggestion
    inputs: InputsEntity[];
    outputs: OutputsEntity[];
    kernels: KernelsEntity[];
```